.github/workflows/check-vcs-installation.yml

```diff
@@ -0,0 +1,74 @@
+name: Check VCS installation
+
+on:
+  push:
+    branches:
+      - main
+      - v*-release
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  check-vcs-installation:
+    runs-on: ubuntu-20.04  # the oldest Ubuntu LTS version
```

Comment:
Maybe also add tests for macOS and Windows. But I think it would be unnecessary. The wheel for `transformers` is platform-independent (the dependencies are platform-dependent).